### PR TITLE
Addresses #2030 Make kidney subset

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Addresses #2030 Make kidney subset (kidney_upper_slim)

### CODE REVIEW

**Review by aleixpuigb — APPROVED** [review]
The subset from [the coverage](https://github.com/obophenotype/cell-ontology/issues/2030#issuecomment-1812636543) is the same than in this PR.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -3188,6 +3190,7 @@ Declaration(AnnotationProperty(oboInOwl:hasSynonymType))
 Declaration(AnnotationProperty(oboInOwl:id))
 Declaration(AnnotationProperty(oboInOwl:inSubset))
 Declaration(AnnotationProperty(oboInOwl:is_inferred))
+Declaration(AnnotationProperty(oboInOwl:seeAlso))
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:3193]: Could you remove this declaration, please?

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -3160,6 +3161,7 @@ Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/cl#added_for_HCA>
 Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/cl#blood_and_immune_upper_slim>))
 Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/cl#eye_upper_slim>))
 Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/cl#general_cell_types_upper_slim>))
+Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/cl#kidney_slim>))
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:3164]: Isn't the slim `<http://purl.obolibrary.org/obo/cl#kidney_upper_slim>`?

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (anitacaron) [reply src/ontology/cl-edit.owl]: Thanks!

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -3244,6 +3246,13 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/cl#general_cell
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/cl#general_cell_types_upper_slim> "general_cell_types_upper_slim")
 SubAnnotationPropertyOf(<http://purl.obolibrary.org/obo/cl#general_cell_types_upper_slim> oboInOwl:SubsetProperty)
 
+# Annotation Property: <http://purl.obolibrary.org/obo/cl#kidney_slim> (kidney_upper_slim)
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:3249]:
```suggestion
# Annotation Property: <http://purl.obolibrary.org/obo/cl#kidney_upper_slim> (kidney_upper_slim)
```

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -3244,6 +3246,13 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/cl#general_cell
...
+
+AnnotationAssertion(<http://purl.org/dc/terms/date> <http://purl.obolibrary.org/obo/cl#kidney_slim> "2023-11-22T11:18:57Z"^^xsd:dateTime)
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:3251]:
```suggestion
AnnotationAssertion(<http://purl.org/dc/terms/date> <http://purl.obolibrary.org/obo/cl#kidney_upper_slim> "2023-11-22T11:18:57Z"^^xsd:dateTime)
```

Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:3251]: Probably this can be fixed automatically and not manually

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (anitacaron) [reply src/ontology/cl-edit.owl:3249]: If refactoring in Protégé, it would change in all other cases.

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -30774,11 +30783,10 @@ EquivalentClasses(obo:CL_4030029 ObjectIntersectionOf(obo:CL_0000542 ObjectSomeV
 
 # Class: obo:CL_4030030 (peripheral blood lymphocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-9990-8331>) Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0003-2534-198X>) Annotation(oboInOwl:hasDbXref <https://sciencing.com/what-peripheral-blood-4672930.html>) Annotation(oboInOwl:hasDbXref "PMID:31092833") obo:IAO_0000115 obo:CL_4030030 "A blood lymphocyte located in the flowing, circulating blood of the body.")
-AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4030030 <https://orcid.org/0000-0002-1773-2692>)
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:30778]: Is there a reason to remove the contributor in this PR?

Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:30778]: It was initially inadvertently added and removed in this PR whilst trying to resolve the QC errors. I'll be doing a larger edit to in a separate PR. If you prefer, I can add it back here, but will be removing in the subsequent PR.

Comment (anitacaron) [reply src/ontology/cl-edit.owl:30778]: Ok, that's fine.

**Comment by bvarner-ebi** [comment]
Thanks for reviewing, @aleixpuigb and @anitacaron!

**Pull request merged**